iOS/Source/InformationButton.swift

```diff
@@ -47,8 +47,8 @@ class InformationButton: UIButton {
         self.nameLabel.heightAnchor.constraint(equalToConstant: 22).isActive = true
     }
 
-    func updateInterface(controller: DaylightModelController) {
-        self.sun.tintColor = controller.highlightColor
-        self.nameLabel.textColor = controller.highlightColor
+    func updateInterface(withDaylightModelController daylightModelController: DaylightModelController) {
```

Comment:
you can probably make this `updateInterface(with daylightModelController:  DaylightModelController)` - in theory someone reading the code would see `updateInterface(with: daylightModelController)` and be able to infer the type pretty easily.